egs/wsj/s5/steps/nnet3/decode.sh

```diff
@@ -32,12 +32,21 @@ extra_left_context_initial=-1
 extra_right_context_final=-1
 online_ivector_dir=
 minimize=false
+word_determinize=true   # If set to false, then output lattice retains alternate paths 
```

Comment:
Sorry to do this at this late stage, but can you please do this instead via copy-and-modify?
I.e. copy decode.sh to steps/nnet3/decode_semisup.sh?  (Or steps/nnet3/semisup/decode.sh if there is anything else that it makes sense to put in a semisup/ directory).

And you can hard-code which ever of these options it makes sense to hard-code.
The rationale is that we don't make the common, simple case more complicated in order to support the obscure case; and this script is seen by a lot of eyes, so adding options to it has a lot of cost.